src/bot.ml

```diff
@@ -130,10 +135,62 @@ let extract_commit json =
         sha )
     |> to_string
 
-let gitlab_ref (issue : GitHub_subscriptions.issue) :
-    GitHub_subscriptions.remote_ref_info =
-  { name= f "pr-%d" issue.number
-  ; repo_url= gitlab_repo ~owner:issue.owner ~name:issue.repo }
+let gitlab_ref ~(issue : GitHub_subscriptions.issue) =
+  let gh_repo = issue.owner ^ "/" ^ issue.repo in
+  let open Lwt.Infix in
+  (* First, we check our hashtable for a key named after the GitHub
+     repository and return the associated GitLab repository. If the
+     key is not found, we load the config file from the default branch.
```

Comment:
We should also document in the README (section "How to use @coqbot instance") about this configuration file.